pom.xml

```diff
@@ -211,6 +211,9 @@ under the License.
                     <groupId>org.apache.avro</groupId>
                     <artifactId>avro-maven-plugin</artifactId>
                     <version>${avro.version}</version>
+                    <configuration>
+                      <stringType>String</stringType>
```

Comment:
This is potentially costly compared to `Utf8` as it uses 2 bytes per char. What's the reason for this change?